src/main/kotlin/no/nav/tilleggsstonader/kontrakter/oppgave/Oppgave.kt

```diff
@@ -9,7 +9,7 @@ data class OppgaveResponse(val oppgaveId: Long)
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 data class Oppgave(
-    val id: Long? = null,
+    val id: Long,
```

Comment:
Bruker vi ikke denne når vi skal opprette oppgaver? Og da må den vel være null? Eller har vi egen type for det?

Reply:
Vi bruker `OpprettOppgaveRequest` når vi oppretter :D

Reply:
👍